src/test/java/org/candlepin/async/tasks/InactiveConsumerCleanerJobTest.java

```diff
@@ -68,14 +75,20 @@ public void testExecutionWithInactiveCheckedInTime() throws JobExecutionExceptio
         inactiveConsumerCleanerJob.execute(context);
 
         consumerCurator.flush();
+        consumerCurator.clear();
+        deletedConsumerCurator.flush();
```

Comment:
Flush and clear are global operations. We only need to do them once, regardless of which curator handles the op.

Reply:
Awesome! I didn't realize they were global operations.